Dockerfile build error

I am trying to build the ompl docker image:

1. Clone this repository
2. Copy ompl.Dockerfile to the root of the repository
3. Build the image: `docker build --file ompl.Dockerfile --tag ompl .`

However I am getting the following error: 
```
291.9 CMake Error at build/cmake_install.cmake:72 (file):
291.9   file INSTALL cannot find "/ompl/build/src/ompl/libompl.so.1.6.0": No such
291.9   file or directory.
291.9 
291.9 
------
ompl.Dockerfile:37
--------------------
  36 |     RUN echo $PATH
  37 | >>> RUN cmake \
  38 | >>>         -G Ninja \
  39 | >>>         -B build \
  40 | >>>         -DPYTHON_EXEC=/usr/bin/python3 \
  41 | >>>         -DOMPL_REGISTRATION=OFF \
  42 | >>>         -DCMAKE_INSTALL_PREFIX=/usr && \
  43 | >>>     cmake --build build -t update_bindings && \
  44 | >>>     cmake --install build && \
  45 | >>>     cd tests/cmake_export && \
  46 | >>>     cmake -B build -DCMAKE_INSTALL_PREFIX=../../install && \
  47 | >>>     cmake --build build
```

Running each of the commands interactively, I found that the error occurs when running `cmake --install build`. I was not able to find the file running `find /ompl/build -name "libompl.so.1.6.0"`.

Any idea why libompl.so.1.6.0 is not being generated? Any help would be much appreciated

--- a/.dockerignore
+++ b/.dockerignore
@@ -2,3 +2,4 @@
 **/*.pyc
 **/*.so
 **/*.dylib
+/build

--- a/scripts/docker/ompl.Dockerfile
+++ b/scripts/docker/ompl.Dockerfile
@@ -41,6 +41,7 @@ RUN cmake \
         -DOMPL_REGISTRATION=OFF \
         -DCMAKE_INSTALL_PREFIX=/usr && \
     cmake --build build -t update_bindings && \
+    cmake --build build && \
     cmake --install build && \
     cd tests/cmake_export && \
     cmake -B build -DCMAKE_INSTALL_PREFIX=../../install && \

